addon/components/ilios-calendar-event.js

```diff
@@ -13,34 +13,85 @@ export default CalendarEvent.extend({
   timeFormat: 'h:mma',
   isDay: false,
   classNameBindings: [':event', ':event-pos', ':ilios-calendar-event', 'isDay:day', 'event.eventClass', 'clickable:clickable'],
+  taughtByPhrase: 'Taught by', // @todo make this translatable. [ST 2016/01/14]
```

Comment:
The calendar defaults to english.  The app that includes it (frontend, or embeded) can pass in a translation.  I don't think these notes add anything to that since it is application wide.  If we want to change that behavior it should just be a ticket.